Exercises/1-seq.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
-const seq = f => g => x => 0;
+const seq = (...f) => a => (typeof a === 'number' ?
+  f.reduce((v, fn) => fn(v), a) :
```

Comment:
We need no reduce here, can solve it just with recursion

Reply:
done. I decided do not use ternary.